front/src/modules/pipeline/constants/pipelineViewFields.tsx

```diff
@@ -17,10 +17,8 @@ import { Entity } from '@/ui/input/relation-picker/types/EntityTypeForSelect';
 export const pipelineViewFields: ViewFieldDefinition<ViewFieldMetadata>[] = [
```

Comment:
maybe not in this PR which is already big, but we need to create a follow up ticket:
this file should be named pipelineFieldsDefinitions.

Reply:
I renamed it `pipelineAvailableFieldDefinitions`.